config.go

```diff
@@ -193,9 +189,6 @@ func NewConfig() *Config {
 	c.Cluster.LongQueryTime = Duration(time.Minute)
 
 	// Gossip config.
-	// c.Gossip.Port = ""
```

Comment:
I had intentionally left these comments in to indicate any setting where we weren't explicitly setting the default value. See also `LogPath: ""`, `c.Cluster.Coordinator = ""` and `c.Metric.Host = ""`. If you think that's unnecessary, then perhaps we should remove those three comments that I mentioned as well.

Reply:
Gotcha, I should have realized those were there as examples. I updated it to add them back and change `Seed` to `Seeds`.